utam-compiler/src/test/java/utam/compiler/grammar/UtamElement_FrameTests.java

```diff
@@ -71,25 +71,6 @@ public void testFrameElementWithNullableThrows() {
                 + "supported are: name, public, selector, type"));
   }
 
-  @Test
-  public void testFrameElementWithElementsThrows() {
```

Comment:
Wouldn't this still be a valid test? According to the docs PR, frames still can't have nested elements. Or did I miss where this is tested elsewhere (a distinct possibility)?

Reply:
refactored in renamed class